app/models/middleware_server.rb

```diff
@@ -5,4 +5,50 @@ class MiddlewareServer < ActiveRecord::Base
   has_many :middleware_deployments, :foreign_key => "server_id", :dependent => :destroy
   serialize :properties
   acts_as_miq_taggable
+
+  include LiveMetricsMixin
+
+  # Used to describe Hawkular supported metrics for this resource and convert it into MiQ style names
+  METRICS_HWK_MIQ = {
+      "WildFly Memory Metrics~Heap Used"                                  => "mw_heap_used",
```

Comment:
Indent only 2 spaces ( @bdunne  what is wrong with @miq-bot...it should have caught this :frowning: )